com.github.git-cola.yml

```diff
@@ -0,0 +1,51 @@
+app-id: com.github.git-cola
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+base: io.qt.qtwebkit.BaseApp
+base-version: "5.15"
+command: git-cola
```

Comment:
you also likely need a `rename-desktop-file` and `rename-appdata-file`, and change the app-dir in the appdata file from upstream.

Note sure about the icons, but likely as well.

Reply:
yeah it needs a `rename-icon: git-cola` as well.

However the appdata doesn't seem to be installed by the build